### ISSUE TITLE
UnboundLocalError: local variable 'is_ignore' referenced before assignment

### ISSUE THREAD

**Issue by SuhasSrinivasan** [uw-ipd/RoseTTAFold2NA#108]
https://github.com/uw-ipd/RoseTTAFold2NA/blob/0e3742355cd48ff1823e2b6a3893b5a07598d16d/input_prep/make_pMSAs_prot_RNA.py#L119-L120

```
Traceback (most recent call last):
  File "/mnt/storage/rf2pip/input_prep/make_pMSAs_prot_RNA.py", line 243, in <module>
    main(fnA, fnB, pair_fn)
  File "/mnt/storage/rf2pip/input_prep/make_pMSAs_prot_RNA.py", line 146, in main
    queryB, a3mB = read_afa(fnB)
  File "/mnt/storage/rf2pip/input_prep/make_pMSAs_prot_RNA.py", line 119, in read_afa
    elif is_ignore:
UnboundLocalError: local variable 'is_ignore' referenced before assignment
```

**Comment by SuhasSrinivasan** [comment]
This error might pop up in the a3m function too

https://github.com/uw-ipd/RoseTTAFold2NA/blob/0e3742355cd48ff1823e2b6a3893b5a07598d16d/input_prep/make_pMSAs_prot_RNA.py#L36


Solution for `read_a3m()` and `read_afa()` is to initialize at Lines 40 and 94
```
is_ignore = True
```

**Comment by SuhasSrinivasan** [comment]
Submitted a Pull Request #111

**Comment by SuhasSrinivasan** [comment]
Fixed in PR #111
